### PR TITLE
32-setup-ci

### LINKED ISSUE #32: Setup CI
The CI should implement:

* javascript linter
* rust linter
* html linter
* css linter

Furthermore the CI should:

* try to build rust

### PR DESCRIPTION
**Pull request by NelDav**

### CODE REVIEW

File: site/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/dist
```
Comment (Szlemens00) [inline site/.gitignore]: A linebreak would be nice :)

File: site/index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
             </div>
             <div class="output_container">
                 <div class="frame">
-                    <label for="frame">Style: </label>
+                    <label for="output_css_selection">Style: </label>
```
Comment (Szlemens00) [inline site/index.html:19]: Should this be outputCssSelection?

File: site/lint/.htmllintrc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "html-req-lang": false,
+    "lang-style": "case",
+    "fig-req-figcaption": false,
+    "focusable-tabindex-style": false,
+    "input-radio-req-name": true,
+    "input-req-label": false,
+    "table-req-caption": false,
+    "table-req-header": false,
+    "tag-req-attr": false
+}
```
Comment (Szlemens00) [inline site/lint/.htmllintrc]: Linebreak

**Review by Szlemens00 — CHANGES_REQUESTED** [review]

File: site/index.html (hunk 2 above)
Comment (NelDav) [reply site/index.html:19]: So far we are using snake_case for ids and classes. I would not mix the naming style.

Comment (Szlemens00) [reply site/index.html:19]: Okay i thought this was a javascript name 👍

**Review by Szlemens00 — APPROVED** [review]

**Pull request merged**